package.json

```diff
@@ -25,6 +25,7 @@
     "test-ci": "npm run lint && npm run build-test && npm run test-remote",
     "test-remote": "./node_modules/karma/bin/karma start --browsers Chrome_travis_ci --single-run",
```

Comment:
just a little `npm` trick:

`npm` adds `node_modules/.bin` folder to the PATH which means any of the dependencies you have that install binaries will be runnable directly. In other words, no need to put "./node_modules/.bin/jshint **.js" or "$(npm bin)/jshint **.js"